examples/aggregator/gcp/pod.yaml

```diff
@@ -24,6 +27,30 @@ spec:
       env:
         - name: RUST_LOG
           value: info
+    # Additional container to scrape prometheus metrics:
+    # https://github.com/GoogleCloudPlatform/k8s-stackdriver/tree/master/prometheus-to-sd
+    - name: prometheus-to-stackdriver
+      image: gcr.io/google-containers/prometheus-to-sd:v0.9.0
+      ports:
+        - containerPort: 6060
+      command:
+        - /monitor
+        # To be able to push custom metrics to the Stackdriver, the `stackdriver-prefix` 
+        # flag must be set to `custom.googleapis.com`.
+        # Ref: https://github.com/GoogleCloudPlatform/k8s-stackdriver/tree/master/prometheus-to-sd#custom-metrics
```

Comment:
Perhaps also add this https://cloud.google.com/monitoring/custom-metrics#about_custom_metrics which is the actual GCP reference.